src/main/java/Eddie/DialogBox.java

```diff
@@ -0,0 +1,50 @@
+package Eddie;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.Node;
```

Comment:
Imports should be listed lexicographically. ie. collections -> geometry -> scene